client/src/Components/Layouts/Login/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { loginValidation } from "../../helper/validationSchema";
+import "./style.css";
+
+export default class Login extends Component {
+  state = {
+    userName: "",
+    password: "",
+  };
+
+  handleChange = ({ target: { name, value } }) => {
+    this.setState({
+      [name]: value.trim()
+    });
+  };
+
+  handleClick = (e) => {
+    e.preventDefault();
+    const { userName, password } = this.state;
+    loginValidation
+      .validate({ userName, password })
```

Comment:
we don't need validation on the login when a username or password wrong  just show message `username or password is incoorect`